remote/src/main/java/com/spotify/metrics/remote/Remote.java

```diff
@@ -30,4 +30,5 @@
  */
 public interface Remote {
     ListenableFuture<Integer> post(String path, String shardKey, Map json);
+    void waitForAllCalls();
```

Comment:
Should also probably be named `shutdown` and prevent future calls to `post` from queuing up more requests.